compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__unknown_variable_update.snap

```diff
@@ -4,10 +4,10 @@ expression: "\npub type Person {\n  Person(name: String, age: Int)\n};\npub fn u
 
 ---
 error: Unknown variable
-  ┌─ /src/one/two.gleam:6:10
+  ┌─ /src/one/two.gleam:6:12
   │
 6 │   Person(..person)
-  │          ^^^^^^^^ did you mean `Person`?
+  │            ^^^^^^ did you mean `Person`?
```

Comment:
These error indicators are now limited to the expression being expanded, which I personally consider a feature :grin: